### PR TITLE
Test coverage improvements

### PR DESCRIPTION
**Pull request by vesin**

### CODE REVIEW

File: api/src/main/java/com/nedap/healthcare/kadasterbagclient/api/dao/AbstractGenericHibernateDao.java [hunk 1]
```diff
@@ -168,21 +167,21 @@ protected T findByCriteriaUnique(final Criterion... criterion) {
...
-     */
-    @SuppressWarnings("unchecked")
-    protected T findUniqueByCriteria(final Criterion... criterion) {
-        final Criteria crit = getCurrentSession().createCriteria(getPersistentClass());
-        for (final Criterion c : criterion) {
-            crit.add(c);
-        }
-        return (T) crit.uniqueResult();
-    }
+    // /**
```
Comment (woutslakhorst) [inline api/src/main/java/com/nedap/healthcare/kadasterbagclient/api/dao/AbstractGenericHibernateDao.java:170]: we have source control, so why checkin code that's commented out?

Comment (vesin) [inline api/src/main/java/com/nedap/healthcare/kadasterbagclient/api/dao/AbstractGenericHibernateDao.java:170]: You are right, it will be deleted.

Comment (woutslakhorst) [inline api/src/main/java/com/nedap/healthcare/kadasterbagclient/api/dao/AbstractGenericHibernateDao.java:170]: Dusko,
[cid:C9FDC134-7893-4922-A826-187951B89748]
Good improvement. The only big red block left is the equals method in Address.java.
Regards,

Comment (cikaRadule) [inline api/src/main/java/com/nedap/healthcare/kadasterbagclient/api/dao/AbstractGenericHibernateDao.java:170]: Common mistake, I forgot to push that part of code.
It's on my home machine, I'll push it when I get back from the office...

**Pull request merged**